content/en/platform/corda/4.11/community/release-notes.md

```diff
@@ -114,6 +114,8 @@ See node migration scripts:
 - `node-core.changelog-v24.xml`: added transaction signatures.
 - `node-core.changelog-v24.xml`: added finality flow recovery metadata.
 
+The `vault_state` table now includes a `consuming_tx_id` column.
```

Comment:
Should the "database changes" be a bulleted list?

Reply:
Yeah, let me format that....